examples/medplum-task-example-app/app/src/pages/NotesPage.tsx

```diff
@@ -0,0 +1,58 @@
+import { Blockquote, Stack } from '@mantine/core';
+import { Annotation, Task } from '@medplum/fhirtypes';
+import { Document } from '@medplum/react';
+
+export interface NotesPageProps {
+  task: Task;
+}
+
+export function NotesPage(props: NotesPageProps): JSX.Element {
+  const notes = props.task.note;
+
+  // Sort notes so the most recent are at the top of the page
+  const sortedNotes = sortNotesByTime(notes);
+
+  // Display if the task does not have any notes
+  if (!sortedNotes) {
+    return (
+      <div>
+        <p>No Notes</p>
+      </div>
+    );
+  }
+
+  return (
+    <Document>
+      <Stack>
+        {sortedNotes.map(
+          (note) =>
+            note.text && (
+              <Blockquote
+                key={`note-${note.text}`}
+                cite={`${note.authorReference?.display || note.authorString} – ${note.time?.slice(0, 10)}`}
+                icon={null}
+              >
+                {note.text}
+              </Blockquote>
```

Comment:
We should probably make htis a core react component

Reply:
There is an existing react component for this in storybook and the react package, but it looks like it isn't an exported component.
https://github.com/medplum/medplum/tree/e129505b926afb939519e088eec2ba97ea39a9e0/packages/react/src/NoteDisplay

Reply:
Can you file an issue to export this?

Reply:
https://github.com/medplum/medplum/issues/3663